bench/f32-gemm-minmax.cc

```diff
@@ -3183,6 +3183,20 @@ static void f32_gemm_minmax_ukernel_4x4__scalar(benchmark::State& state, const c
 
 BENCHMARK_GEMM(f32_gemm_minmax_ukernel_4x4__scalar)
 
+#if XNN_ENABLE_RISCV_VECTOR && XNN_ARCH_RISCV
+  static void f32_gemm_minmax_ukernel_7x4v__rvv(benchmark::State& state, const char* net) {
+    GEMMBenchmark(state,
+      xnn_f32_gemm_minmax_ukernel_7x4v__rvv,
+      xnn_init_f32_minmax_scalar_params,
+      xnn_pack_f32_gemm_goi_w,
+      /*mr=*/7, /*nr=*/4 * xnn_init_hardware_config()->vlenb / sizeof(float), /*kr=*/1, /*sr=*/1,
```

Comment:
This is generated, not manually added, right?
The generator knows about vlen?

Reply:
This isn't manually added.
The generator knows about `vlen` because of modifications in `tools/generate-gemm-test.py`.